data/can_prov_cases.R

```diff
@@ -0,0 +1,8 @@
+delayedAssign("can_prov_cases", local({
```

Comment:
note: I think these delayedAssigns could all move to R/, and could be roxygen-documented directly as if they were ordinary objects (roxygen2 should detect delayedAssign, but did not detect our own variant in epiprocess).  By migrating
```
data/can_prov_cases.R
cases_deaths_subset.R
counts_subset.R
covid_case_death_rates.R
covid_incidence_county_subset.R
covid_incidence_outliers.R
```
into R/
we could also remove
```
data/can_prov_cases.rda
data/cases_deaths_subset.rda
data/counts_subset.rda
data/covid_case_death_rates.rda
data/covid_incidence_county_subset.rda
data/covid_incidence_outliers.rda
```
I also don't understand the sizes of these current .rda files... e.g. covid_case_death_rates.rda at 368K does not seem like it's just storing the promise.

Reply:
I left files in `data` so that the data is accessible in the normal `data(...)` way. I figured it would be more familiar to users.

I think those `*.rda` files are local to you. On size, i think they're storing the actual data, not just a promise.